<?xml version="1.0" encoding="utf-8"?>
<launch>

  <node name="rviz" pkg="rviz" type="rviz" output="screen"
    args="-d $(find gcopter)/config/global_planning.rviz"> 
  </node>
  
  <!-- <node name="pos_vel_plot" pkg="rqt_plot" type="rqt_plot" 
    args="/visualizer/speed /visualizer/total_thrust /visualizer/tilt_angle /visualizer/body_rate">
  </node> -->

  <node pkg="polys_mapgen" type="hpolys_mapgen_node" name="hpolys_mapgen_node" output="screen">
  <rosparam file="$(find polys_mapgen)/config/mapgen.yaml" command="load" />
  <remap from="/polys_pc_map" to="/voxel_map"/>
  </node>
  
  <node pkg="gcopter" type="global_planning" name="global_planning_node" output="screen">
  <rosparam file="$(find gcopter)/config/global_planning.yaml" command="load" />
  </node>

</launch>
